freqtrade/tests/test_fiat_convert.py

```diff
@@ -126,6 +126,13 @@ def test_fiat_convert_get_price(mocker):
     assert fiat_convert._pairs[0]._expiration is not expiration
 
 
+def test_fiat_convert_same_currencies(mocker):
```

Comment:
would a test for EUR / USD make sense too? (currently not sure what that would return as coinmarketcap does not support that ... but that would make a test even more interresting...)

Reply:
I will add the test, but it will fall in the next exception that returns 0.0.

Reply:
good point, then it probably doesn't make sense to test for it

Reply:
Better to test than making assumptions :)